cypress/e2e/onboarding.ts

```diff
@@ -30,15 +30,18 @@ describe('onboarding', () => {
     cy.findByRole('main').within(() => {
       cy.findByRole('textbox', {name: /name/i}).type(firstName)
       cy.findByRole('group', {name: /team/i}).within(() => {
-        cy.findByRole('radio', {name: /blue/i}).click()
+        // checkbox is covered with a <label>
+        cy.findByRole('radio', {name: /blue/i}).click({force: true})
       })
-      cy.findByRole('button', {name: /join KCD/i}).click()
+      cy.findByRole('button', {name: /create account/i}).click()
 
-      cy.findByRole('button', {name: /logout/i}).click()
+      // TODO: find a new home for the logout button?
+      // cy.findByRole('button', {name: /logout/i}).click()
     })
 
-    cy.findByRole('navigation').within(() => {
-      cy.findByRole('link', {name: /login/i})
-    })
+    // TODO: find a new home for the login-with-qr-code thing?
+    // cy.findByRole('navigation').within(() => {
+    //   cy.findByRole('link', {name: /login/i})
+    // })
```

Comment:
Ok. These are not so awesome. But the current designs haven't taken them into account, and I lack the creativity to come up with something nice. I'll ask Gil.

Reply:
Oh wait, I'm mixing stuff up, aren't I. This is `login link`, shown on the signup page, not on the `/me` page. So that's not the QR. What is it?

Reply:
aah, I'm mixing something up. This should be the main nav link, right? Let's fix that one.